src/reducers/query/querySlice.js

```diff
@@ -131,27 +148,24 @@ export const querySlice = createSlice({
           : null;
 
         const datesChanged =
-          state[fields[0]] !== minDate || state[fields[1]] !== maxDate;
+          state.date_received_min !== minDate ||
```

Comment:
is there a missing expression after the `||` clause, or is it line 152?

Reply:
it's line 152. 

the autoformatter puts it on the next line due to line length.

Reply:
Got it, thanks!